### PR TITLE
Use Ext4 locators

### PR DESCRIPTION
**Pull request by labkey-adam**
#### Rationale
Update test to match requestability rules page Ext3 -> Ext4 conversation

#### Related Pull Requests
* https://github.com/LabKey/platform/pull/4959

### CODE REVIEW

File: data/api/executeSql.html [hunk 1]
```diff
@@ -2,25 +2,28 @@
 <div><input id="qwpSchema"/></div>
 <div><label for="qwpSql">SQL:</label></div>
 <div><textarea rows="10" cols="65" id="qwpSql"></textarea></div>
-<div><input type="button" value="Execute SQL" onClick="loadQWP();" id="button_loadqwp"/></div>
+<div><input type="button" value="Execute SQL" id="button_loadqwp"/></div>
 <div id="qwp-div">Query Web Part DIV</div>
 
 <script type="text/javascript">
```
Comment (labkey-tchad) [inline data/api/executeSql.html:8]: Script needs nonce

File: src/org/labkey/test/tests/SpecimenBaseTest.java [hunk 2]
```diff
@@ -60,19 +63,28 @@ protected void setupRequestabilityRules()
...
+        // Expect three rows
+        int rowCount = gridRef.getRowCount();
+        log("How many rows? " + rowCount);
+        Assert.assertEquals(3, rowCount);
+
         // Verify that LOCKED_IN_REQUEST is the last rule
-        waitForElement(Locator.xpath("//div[contains(@class, 'x-grid3-row-last')]//div[text()='Locked In Request Check']"));
-        click(Locator.xpath("//div[contains(@class, 'x-grid3-row-last')]//div[text()='Locked In Request Check']"));
+        log(gridRef.getFieldValue(rowCount, "name").toString());
+        Assert.assertEquals("Locked In Request Check", gridRef.getFieldValue(rowCount, "name"));
```
Comment (labkey-tchad) [inline src/org/labkey/test/tests/SpecimenBaseTest.java]: Assertions should have a message.
```suggestion
        Assert.assertEquals("Last requestability rule", "Locked In Request Check", gridRef.getFieldValue(rowCount, "name"));
```

**Review by labkey-tchad — APPROVED** [review]

File: data/api/executeSql.html (hunk 1 above)
Comment (labkey-adam) [reply data/api/executeSql.html:8]: > Script needs nonce

I believe this is pulled into a wiki, so it does **not** require a nonce. Wikis are a special case: LabKey always parses them and adds the `nonce` element to the DOM.

Comment (labkey-tchad) [reply data/api/executeSql.html:8]: Oh, yup. I missed the context.

**Pull request merged**